simulariumio/data_objects/unit_data.py

```diff
@@ -63,6 +65,23 @@ def multiply(self, multiplier: float):
         self._quantity *= multiplier
         self._update_units()
 
+    @classmethod
+    def from_dict(
+        cls,
+        unit_data: Dict[str, Any],
+        default_name: str = None,
```

Comment:
just wondering why the choice of None for default_name as opposed to an actual value?

Reply:
I think I see, maybe it's because you don't know if the default unit should be space or time?  I wonder if you can just make it required and not accept a default.

Reply:
Yeah so the case I was trying to handle is if someone's filling out the form to convert their data to simularium file format, they don't change the value that's visible in the dropdown for unit name, and thus we'd want to fill that with whatever the form defaulted to showing. So, requiring it would work well for that desired use case / what `SmoldynData` is doing, since I know what I want the default values to be for smoldyn autoconversion.

The problem is, I decided to use `UnitData.fromDict()` in `TrajectoryData.from_buffer_data()` too and I have no idea what to put as the default time units and spatial units there. Perhaps I just shouldn't use it in `trajectory_data.py`? Or I should make up some default values? Open to suggestion, I'm not sure what's best

Reply:
In `trajectory_data.py` you can use "s" for default time unit and "m" for default spatial unit. Better yet, add those to constants and import them in `trajectory_data.py`